citi-onboarding/HighBall
# Purpose

## What I did

- Created puirpose section
   
## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout purpose`
   1. `git pull origin purpose`
   1. `yarn install`
   1. `code .`
   1. `cd client`
   1. `yarn start`

1. In your browser go to http://localhost:3000

1. In VScode open `./client`

1. Read the changed files section on GitHub 

1. Compare the page at 1366px width with the [mockup](https://www.figma.com/file/FTlYITKEgwlp7rDJzX6IBc/Squad-Julia?node-id=189%3A6)

1. Check if the background blob expands as the screen size is resized

1. Test the things that you think are worth testing, even the ones that are not in this description
